File: src/backend/SyllabusManager.Data/Models/Subjects/LearningOutcomeEvaluation.cs
using SyllabusManager.Data.Enums.Subjects;
using System.ComponentModel.DataAnnotations;

namespace SyllabusManager.Data.Models.Subjects
{
    public class LearningOutcomeEvaluation : ModelBase
    {
        [Required]
        public GradingSystem GradingSystem { get; set; }
        [Required]
        public string LearningOutcomeSymbol { get; set; }
        public string Description { get; set; }
    }
}

File: src/frontend/src/app/core/models/subject/learning-outcome-evaluation.ts
import { GradingSystem } from "../../enums/subject/grading-system.enum";
import { ModelBase } from "../model-base";

export class LearningOutcomeEvaluation extends ModelBase {
    gradingSystem: GradingSystem = GradingSystem.DuringSemester;
    learningOutcomeSymbol: string = '';
    description?: string;
}
